pwiz_tools/Skyline/Util/Xml.cs

```diff
@@ -961,6 +961,18 @@ public static string UnescapeNonPrintableChars(this string str)
             return sb.ToString();
         }
 
+        // Inspect a file path for characters that must be escaped for use in XML (currently just "&")

```

Comment:
I could imagine putting this on PathEx also as EscapePathForXml() with EscapePathForCommandLine()... Or moving EscapePathForCommandLine() to ProcessRunner.EscapePath(). Feels unbalanced the way it is.

Reply:
I can do that, sure.